src/lib/moment/start-end-of.js

```diff
@@ -1,59 +1,127 @@
 import { normalizeUnits } from '../units/aliases';
+import { hooks } from '../utils/hooks';
+
+var MS_PER_SECOND = 1000;
+var MS_PER_MINUTE = 60 * MS_PER_SECOND;
+var MS_PER_HOUR = 60 * MS_PER_MINUTE;
+var MS_PER_24_HOUR_DAY = 24 * MS_PER_HOUR;
```

Comment:
use MS_PER_DAY

Reply:
That was a deliberate name choice.  DST means 'MS_PER_DAY' would be a misnomer.